products/sql/examples/ansible/user.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Provider::Ansible::Example
+dependencies:
+  - !ruby/object:Provider::Ansible::Task
+    name: gcp_sql_instance
+    code: |
+      name: <%= dependency_name('instance', 'user') %>
```

Comment:
nit: We're supposed to use `-%>` everywhere, right?

Reply:
Yeah. This is a bug that I want to address as part of a bigger rewrite of examples.